cves/CVE-2012-6617.yml

```diff
@@ -127,7 +125,7 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer:  This line of code was created in 2007-08-20 and was fixed on 2012-12-09. However, the class and the subsystem, ffserver, was depreciated in FFmpeg 4.0 (Wu) on 2018-04-20
```

Comment:
I know it's not _super_ related to the vulnerability, but do you know why it was depreciated?

Reply:
It was not maintained.